### ISSUE TITLE
Set up paid-advertising to gain attention

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdropcms.org#124]
Places we should advertise:
- goolge
- facebook
- other?

**Comment by Gormartsen** [comment]
Hi Guys!

I can help to advertise backdrop on Drupal.ru

**Comment by jenlampton** [comment]
@Gormartsen that would be fantastic!  What is involved in getting that set up?

**Comment by Gormartsen** [comment]
It depends what we want.
I can propose next things:
- make an interview with core developers with most hot questions
- start to announce online meetups
- initiate translation initiative
- make a demo website
- make a promo page in russian with feature list

That's what I have in my mind right now.

It's good to have marketing policy to keep all this promotions in line with it.

**Comment by sutibun** [comment]
Here is what I roughly had in mind.

We start with Facebook because of its ability to target specific types of people. (read: Drupal users)

Here is a theory I have:

- The most active and vocal Drupal users you see on Drupal.org and its forums are the most "advanced" and techie of users. (let's peg it at 10% of the total Drupal community)
- The other 90% of Drupal users, I think, are less advanced and you don't hear from them as much but they're there.

I think the 10% advanced users will likely upgrade with Drupal 8 because they know how and have the skills. The rest of the Drupal 90% users aren't likely (like the monkey). They're probably on the fence. Most likely they want to continue with Drupal but v8 is a giant leap. I suspect, though have nothing to based this on, that these 90% haven't even heard of Bd at all like I did months before. They'd be more receptive to switching to Bd.

Anyway, point being these more normal users are likely using FB the most. At least more than the advanced users.

Now very roughly what I was planning to do is: 

1. Create a blog post that talks about the "fall" of Drupal 8 (and introduces Bd)
2. Publish link on FB
3. Pay FB to show this strategically created content to only normal Drupal users.
4. Have current Bd fans like it and share it to give extra boost.

In the end, hoping that the Drupal users on FB who read it will like it and share it as well... thus hopefully spreading even beyond what we paid for.

I'll have to check the details but I *think*, don't quote me on this, we can start with FB advertising with about $50-$100. We can use it to test the waters too.

Advertising is all about testing. We shouldn't just pay lots of money on one shot. That's a recipe for losing money quick. This isn't brand advertising.... this is meant to strategically go after a specific type of user.

**Comment by Gormartsen** [comment]
Better to do not say 'Drupal 8 fall' in a negative sense. 
Let's put - moving towards big business solutions ( enterprise?) ? 
I mean post should not focus on competition, but benefits instead. Like easy to move modules, new layout concept that help to control website look and feel more precisely , mobile friendly and etc

**Comment by sutibun** [comment]
"Fall" ... hehe people like controversy. Advertising is all about grabbing attention.

If your post isn't interesting, people won't read it. Plus, the monkey is all about attention.

But guess can tone it down if everyone agrees.

**Comment by klonos** [comment]
Yeah, I would not get too "aggressive" with D8. We should be more "D8 too hard for you? Then give BD a go - it's more like D7 with D8 features added!" rather than "D8 failed/sucks!!! BD IS AWSOME!!". ...if you get my drift.

**Comment by sutibun** [comment]
but monkey so wanna bash something... ^\_^

Anyway, to briefly summarize my initial plan for any paid advertising, it is this: I am trying to replicate the way I first learned of Backdrop. https://github.com/backdrop-ops/backdropcms.org/issues/122#issue-118232987 by presenting a blog post to other Drupal users which introduces Bd. 

The reason I suggest FB is because if it resonates, those same people might Like and Share it, which will help spread it to even more people for free.

That's the initial plan to test out.

**Comment by klonos** [comment]
Yes, in general what the Monkey suggests makes sense.

**Comment by jenlampton** [comment]
I set up some google ad campaigns for pricing, and it's clear we can't afford to market using the keyword "Drupal" since all the big players are throwing a lot of money at that keyword. 

So I started looking at more specific targeting, things like "Drupal 7 end-of-life" and "Drupal alternative" and "Drupal competitor" "Drupal migration" "Drupal upgrade". 

We could hit these kinds of keywords for $100-200/mo

**Comment by oadaeh** [comment]
We should also create pages specifically targeting those search terms so that we show up organically for those queries. Far more people click on organic results than paid results, something like 60% vs ~7%, so making sure the organic result is taken care of is important.

**Comment by jenlampton** [comment]
That's a great idea @oadaeh

**Comment by laryn** [comment]
Cross referencing: https://github.com/backdrop-ops/backdropcms.org/issues/542
